src/tools/cephfs/cephfs-shell

```diff
@@ -718,6 +717,16 @@ class CephFSShell(Cmd):
         """
         Copy a file/directory from CephFS to given path.
         """
+        if not is_file_exists(args.remote_path):
+            set_exit_code_msg(errno.ENOENT, 'error: no file found at '
+                                            'specified remote path')
+            return
+        if os.path.exists(args.local_path) and not args.force:
+            set_exit_code_msg(msg=f"error: file "
```

Comment:
We won't be getting proper error message whether `file with same name already exists` or `destination_path already exists`.It will only show message as `dest_path` already exists in both cases.I guess, that's why earlier we had two checks,for both cases.

Reply:
I get the point that it doesn't tell if the path is absent or the file, I think `isfile()` should replace `exists()`. I hope having a destination path already is not at all a problem, right?

Reply:
Check now